src/service/xhr-impl.js

```diff
@@ -534,6 +535,14 @@ class FetchResponseHeaders {
   get(name) {
     return this.xhr_.getResponseHeader(name);
   }
+
+  /**
+   * @param {string} name
+   * @return {boolean}
+   */
+  has(name) {
```

Comment:
It is used in google/ads/units/a4a.js (and the root cause of https://github.com/ampproject/amphtml/issues/6160 bug)